### ISSUE TITLE
Add ability to lock sections and/or blocks from being editable.

### ISSUE THREAD

**Issue by aaronware** [linchpinagency/mesh#50]
When you create a template with sections and blocks have the ability to completely lock the editing of sections and blocks when the template is being used as a "reference" template.

**Comment by strarsis** [comment]
@aaronware: Does this also mean that the number + "ratio" of columns/sections can be locked down?
I have a template that got two columns in specific ratio - I want to reflect this in backend - using Mesh plugin. I don't want the user to change the number of columns/sections and ratios.

**Comment by aaronware** [comment]
@strarsis yes. The plan is basically to lock down any "visual" or "layout" specific elements to the template. So it will remove controls to change those elements. The plan is to also mass update any pages/posts that use the selected template. The last portion that i just mentioned is what the holdup is. I have everything in place except an efficient way to update any pages using said template
